Dockerfile

```diff
@@ -0,0 +1,12 @@
+#Build stage
+FROM gradle:jdk17-corretto AS build
+WORKDIR /usr/app/
+COPY . .
+RUN gradle build -x test
```

Comment:
Use --no-daemon in Gradle build command

Prevents Gradle from starting its background daemon, reducing memory usage and potential errors in containerized environments.